### DISCUSSION TITLE
Lost superadmin permissions

### DISCUSSION THREAD

**Discussion by TheLittleLebowsk** [Pryaxis/TShock#2045] [Q&A]
I wanted to set up a group with the permission to use Pylons to teleport. To test if the group worked I added myself to it and now I can't use any commands except the basic commands everyone can use like /help or /rules and the tp commands i added to the group. When I try to use any other command I just get the message "You do not have access to this command".
How do I regain access to the superadmin commands?

**Comment by hakusaro** [comment]
From your console, run `/user add newuser newpassword superadmin` where `newuser` is your new username and `newpassword` is your new password, and then use `/login newuser newpassword` to be superadmin again.
